### PR TITLE
Add support for ASM230. Remove support for ASM228.

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

This pull request does:
* add support for ASM230
* remove support for ASM228
* remove old obsoleted preload methods (we are moving to 3.0)
* remove old libG version mapping

### Declarations

Check these if you believe they are true

- [X ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [X] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [X] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Tools/DynamoShapeManager/Preloader.cs [hunk 1]
```diff
@@ -144,60 +97,6 @@ public Preloader(string rootFolder, IEnumerable<Version> versions)
...
-        /// core modules. This must represent a valid directory.
-        /// </param>
-        /// <param name="version">The version of shape manager.</param>
-        /// <returns></returns>
-        [Obsolete("please use constructor Preloader constructor with signature Preloader(string,IEnumerable<Version>)")]
-#if NET6_0_OR_GREATER
-        [System.Runtime.Versioning.SupportedOSPlatform("windows")]
-#endif
-        public Preloader(string rootFolder, LibraryVersion version)
-            : this(rootFolder, new[] { version }) { }
```
Comment (QilongTang) [inline src/Tools/DynamoShapeManager/Preloader.cs:200]: Forgive me to ask. Just curious why are we removing so much code here.

Comment (sm6srw) [reply src/Tools/DynamoShapeManager/Preloader.cs:200]: They are obsolete and we are moving to 3.0.

File: tools/install/Extra/additional-files-to-sign.txt [hunk 2]
```diff
@@ -6,15 +6,6 @@
 Greg.dll
 Analytics.Net.ADP.dll
 Analytics.NET.Core.dll
-LibG.Interface.dll
 ProtoGeometry.customization.dll
-ProtoGeometry.dll
```
Comment (mjkkirschner) [inline tools/install/Extra/additional-files-to-sign.txt:11]: what are these changes about?

Comment (sm6srw) [reply tools/install/Extra/additional-files-to-sign.txt:11]: Yes, they are already signed as part of the LibG build.

**Pull request merged**